### ISSUE TITLE
Missing draw:editmoved event

### ISSUE THREAD

**Issue by marcellangmaier** [Leaflet/Leaflet.draw#699]
I am looking for an event when the user stops moving around a feature. (dropping it)
Then I want to callback the new position to the server.

Same thing with the draw:editresize. A draw:editresized event would be perfect.

**Comment by germanjoey** [comment]
This is in the PR I sent in 3 months ago. If you can figure out a way to
get @ddproxy to do something about it, please do, because I don't know what
else I can do.

**Comment by alexandervlpl** [comment]
Yes, exactly what I need! Two new events (`editmoved` and `editresized`) which fire when a user is done editing a feature but hasn't saved it yet.

The current events fire way too much, you can forget about doing an AJAX request or anything complex in the handler functions.

**Comment by alexandervlpl** [comment]
Here's my workaround:

```
var now_editing = false;

map.on('draw:editstart', function (e) {
    now_editing = true;
});    

map.on('draw:editstop', function (e) {
    now_editing = false;
});
    
map.on('mouseup', function (e) {
    if (now_editing) {
        // Do something here.
    }
});
```

Obviously, `mouseup` will trigger when the user just moved the map or clicks on it (without actually changing the drawn features). There's also no way to differentiate between resize and move. Works for me though.
